lib/rsmp/supervisor.rb

```diff
@@ -107,10 +108,6 @@ def accept? socket, info
       true
     end
 
-    def build_proxy settings
```

Comment:
I think we should keep this method, and construct the proxy there

Reply:
yes by i want the code that construct the proxy moved to a separate method build_proxy(), as it was before